### PR TITLE
fix(iam): use access_token for iam

### PR DESCRIPTION
**Pull request by mamoonraja**

### CODE REVIEW

File: views/demo.jsx [hunk 1]
```diff
@@ -76,6 +76,7 @@ export default React.createClass({
     const keywords = this.getKeywordsArrUnique();
     return Object.assign({
       // formats phone numbers, currency, etc. (server-side)
+      access_token: this.state.access_token,
```
Comment (germanattanasio) [inline views/demo.jsx]:
```js
access_token: this.state.accessToken
```

File: app.js [hunk 2]
```diff
@@ -51,10 +54,19 @@ app.get('/api/credentials', (req, res, next) => {
     if (err) {
       next(err);
     } else {
-      res.json({
-        token,
```
Comment (germanattanasio) [inline app.js]:
```js
res.json({
  token,
  serviceUrl,
  accessToken,
});
```

Comment (mamoonraja) [reply app.js]: need to pass either/or not both of them.

**Comment by germanattanasio** [comment]
Fixes #205

**Pull request merged**